bbc_clean/treerez/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.24.2
```

Comment:
Any reason to lock to specific versions for this simple tool?

Reply:
Python module dependencies can be quite finicky... I would rather be explicit about versions we know for sure work happily together.